lib/index.js

```diff
@@ -365,6 +343,24 @@ function stringToBuffer(string) {
 }
 
 /**
+ *
+ * Converts an ArrayBuffer to a Buffer
+ *
+ * @api private
+ */
+
+function arrayBufferToBuffer(data) {
+  // data is either an ArrayBuffer or ArrayBufferView.
+  var array = new Uint8Array(data.buffer || data),
+    buffer = new Buffer(array.length);
+
+  for (var i = 0; i < array.length; ++i) {
```

Comment:
We don't do `++1`. This doesn't affect performance too. http://stackoverflow.com/questions/1546981/post-increment-vs-pre-increment-javascript-optimization